homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,44 @@
+const tamagochi = {
+    name: 'Tamagochi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
+    noEating() {
+        tamagochi.healthLevels -= 1;
+        tamagochi.happyLevels -= 1;
+        tamagochi.foodLevels -= 1;
+
+        const tamagochiStates = `Здоров'я  в ${tamagochi.name} - ${tamagochi.healthLevels} Щастя - ${tamagochi.happyLevels} Ситість - ${tamagochi.foodLevels}`;
+
+        const tamagochiDiedFirst = tamagochi.healthLevels === 0 || tamagochi.happyLevels === 0;
+        const tamagochiDiedSecond = tamagochi.foodLevels === 0;
```

Comment:
I don't quite understand the meaning of the `tamagochiDiedFirst` and `tamagochiDiedSecond` variable names. I suggest you create more descriptive reasons for the dead.